packages/core/src/services/tracing/spans/processBulk.ts

```diff
@@ -364,6 +364,10 @@ export async function processSpansBulk(
           'projectId' in processed.metadata
             ? (processed.metadata.projectId as number)
             : undefined,
+        previousSpanId:
+          'previousSpanId' in processed.metadata
+            ? (processed.metadata.previousSpanId as string)
+            : undefined,
```

Comment:
Good opportunity to add some tests here. This service is ultra core and is 0 tested